tmdb/src/main/java/me/banes/chris/tivi/tmdb/TmdbImageUrlProvider.kt

```diff
@@ -62,4 +55,8 @@ class TmdbImageUrlProvider(
 
         return previousSize ?: sizes.last()
     }
+
+    private fun extractWidthAsIntFrom(size: String): Int? {
+        return "w(\\d+)$".toRegex().matchEntire(size)?.groups?.get(1)?.value?.toInt()
```

Comment:
- `w` is the literal
- `\d+` is 1 or more instances of a digit
- `$` is end of string, to disallow identifiers like `w100foo`, only accepting format `w100`

would have extracted to a `private static final String` in Java, but not sure of the equivalent in Kotlin 🤔

Reply:
>would have extracted to a private static final String in Java, but not sure of the equivalent in Kotlin 

You could use `private const val PATTERN = "w(\\d+)$"`

Reply:
perfect thanks! 

commit 9a1617908386639823be731ba45fdc93894de34d
Author: Ataul Munim <hello@ataulm.com>
Date:   Sun Jan 14 12:56:02 2018 +0000

    Extract regex pattern to private const